### DISCUSSION TITLE
Consistency v. Variety

### DISCUSSION THREAD

**Discussion by jasonjgardner** [jasonjgardner/jg-rtx#55] [Feedback]
### Which do you prefer in Minecraft textures?

(Specifically, what kind of designs to you want or expect to see in JG RTX?)

#### Consistent Designs
Textures which vary only in hue, such as the _vanilla_ design of concretes, wools, stained glass, etc.
![Stained glass consistency](https://user-images.githubusercontent.com/1903667/148662725-df71495a-9522-4094-8f5b-30423078f17d.png)

#### Design Varieties
Textures which have more unique or individual designs, such as the vanilla design of bricks, glazed terracotta, etc.
![Stained glass variety](https://user-images.githubusercontent.com/1903667/148662739-99c12178-3299-43b0-b8bd-71c6dceb9af1.png)

---

### Where does it matter?
- Which blocks _must_ be consistent?
- Which blocks _must_ have a variety?
- Where is a good to have a variety?
- Where is it bad to have a variety?

**Comment by erich666** [comment]
I haven't thought about it all that much, I admit. I like variety for the "decorator blocks," for sure - terracotta. For wool I'd want just the color to vary. Glass I like some variation, as I've mentioned. Things like the various ores and stones and sands, they should have the same overall texture (stone, say) with the variations added.

So, no deep observations from me; probably pretty much what you might guess.
